### PR TITLE
Add qualifier on Array spread with NodeLists

### PR DESCRIPTION
**Pull request by Josh68**
Transpiling for IE11 seems to break this method

/cc @gerardo-rodriguez

### CODE REVIEW

File: javascript/README.md [hunk 1]
```diff
@@ -426,6 +426,8 @@ const animalsCopy = [...animals];
 
 To convert an iterable object (e.g. [NodeList]) to an array, use [array spread syntax][Array Literal Spread Syntax] `...` instead of [Array.from].
 
+(Note: This may not work correctly if transpiling with support for IE11. Instead, try a `for ... of` loop over the `NodeList`.)
```
Comment (tylersticka) [inline javascript/README.md]: Is there an article, issue or StackOverflow post we could link to that might describe this in more detail? I've used this trick on other projects and haven't noticed IE11 regressions (though I _have_ configured Babel incorrectly and seen this issue)

Comment (Josh68) [reply javascript/README.md]: I got to the heart of this, and it's not what I thought originally. https://github.com/cloudfour/guides/issues/52.  Revising.

Comment (Josh68) [reply javascript/README.md]: Revised. /cc @gerardo-rodriguez @tylersticka @spaceninja @calebeby @Paul-Hebert

**Comment by spaceninja** [comment]
Hey @cloudfour/dev team any further objections to this, or can we approve @Josh68's PR?

**Pull request merged**